app/src/main/res/layout-land/topic_info_fragment.xml

```diff
@@ -20,14 +20,14 @@
       android:layout_width="match_parent"
       android:layout_height="wrap_content"
       android:paddingTop="@dimen/padding_24dp"
-      android:paddingBottom="24dp">
+      android:paddingBottom="@dimen/padding_24dp">
 
       <TextView
         android:id="@+id/topic_name_text_view"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:layout_marginStart="32dp"
-        android:layout_marginEnd="32dp"
+        android:layout_marginStart="72dp"
+        android:layout_marginEnd="72dp"
```

Comment:
on 35 line number there is constraint given with respect to image. Please remove that line it is no more required.

Reply:
if i remove that the text view wont have any horizontal constraint

Reply:
You can give top constraint.